src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -316,6 +316,14 @@ internal bool ServerMode
             }
         }
 
+        internal bool ShowVersion
+        {
+            get
+            {
+                return _showVersion;
```

Comment:
I think it's fine to follow the existing code pattern here. We can convert the style using a tool later.